modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -0,0 +1,397 @@
+----
+Activating the Kiteworks satellite ....
+Verifying users ...
+No Email for user newuser - it cannot be migrated to Kiteworks!
+Please make sure all users meet the requirements.
+This instance is NOT ready to be migrated to Kiteworks!
+----
+
+=== Migration
+
+After all prerequisites, installations, configurations and the verification has passed, you can initiate the migration process. For a possible improved transfer performace, read the xref:tuning-transfer-performance[Tuning Transfer Performance] first. Issue the following command to start the migration:
+
+[source,bash]
+----
+sudo -u www-data \
+  php /var/www/owncloud/occ \
+  migrate:to-kiteworks \
+  $KW_ADMIN_USER
+----
+
+During the migration process, a log file named `migrate-kiteworks-<timestamp>.csv` is created in the ownCloud root folder. This file contains:
+
+* general rclone responses and errors,
+* rclone responses for user migration,
+* rclone casing conflicts that an ownCloud admin must solve.
+
+Any transfer that can be processed by rclone will finish.
+
+.Example for migration issues reported
+[source,plaintext]
+----
+Issues did arise when migrating files and folders..
+Please review migrate-kiteworks-<timestamp>.csv and fix any issues which have been reported.
+
+Once resolved please re-run the migration process again.
+
+Migration will stop here now until no more conflicts exist.
+----
+
+.Examples for casing conflicts noted in the migration log file:
+[source,plaintext]
+----
+NOTICE,user1,user1@example.com,"2024/04/03 15:20:27
+  NOTICE: Photos: Duplicate directory found in source - ignoring"
+
+NOTICE,user2,user2@example.com,"2024/04/03 15:20:32
+  NOTICE: Documents/Example.odt: Duplicate object found in source - ignoring"
+----
+
+As you can see above, there is a casing conflict for a file and a directory. The conflict takes place because a file or directory has been migrated already conflicting with the name of the reported object. The conflicts for the particular users need to be resolved. If this is done, the migration can be restarted and only those objetcs that have not been migrated get processed.
+
+==== Solving Casing Conflicts
+
+If there are casing conflicts reported in the shell respectively in the migration log, the ownCloud admin must solve them to continue the migration. 
+
+For reported conflicts, the admin should impersonate as the user with the conflict and solve it by renaming the file or directory according the Kiteworks namig rules. Post fixing all open casing issues, the migration can be restarted and all formerly conflicted files or folders will get migrated.
+
+Note that a migration restart will only migrate objects that have not been migrated so far.
+
+== Tuning Transfer Performance
+
+Per default, rclone transfers 4 files in parallel. This creates little load on the target system, but may take a longer time to complete. This is especially true when anticipating mostly small files like 10k instead of large files like 10 MB or above.
```

Comment:
```suggestion
By default, rclone transfers 4 files in parallel. This creates little load on the target system, but may take a longer time to complete. This is especially true when anticipating mostly small files with sizes of about 10KB instead of large files with sizes of 10 MB or above.
```